doc/source/api/index.rst

```diff
@@ -43,6 +43,14 @@ system for reads and variants.
 .. toctree::
    references
 
+Sequence Annotations
```

Comment:
"Sequence Features" seems more descriptive (and avoids the well-worn term "annotations").